docs/migration/css-changes-in-version-4.md

```diff
@@ -5,3 +5,85 @@ title: Changes to Styles in 4.0
+
+* `cx-configurator-image` mixin has been defined on `%cx-configurator-product-title` for `cx-details-image` selector to define the unique styling.
+
+* `cx-configurator-truncate-content` mixin has been added on `%cx-configurator-product-title` for `cx-detail-title`, `cx-code` and `cx-description` selectors to enable the truncation for the small widgets.
+
+## Change in Configurator Group Menu Component
+
+* `cx-configurator-truncate-content` mixin has been added on `%cx-configurator-group-menu` for `span` selector to enable the configuration group title truncation for the small widgets.
+
+## Changes in Configurator Form Component
+
+* `width` set to 100% on `%cx-configurator-form` to use the whole width of the configuration form.
+
+* `padding` set to 16px on `%cx-configurator-form` for `cx-group-attribute` to align the spacing between the configuration group attributes.
+
+## Changes in Configurator Attribute Header Component
+
+* `margin` set to 17px on `%cx-configurator-attribute-header` to align the spacing to the attribute header to the attribute type.
+
+* `padding` set to 0px and `margin` to 17px on `%cx-configurator-attribute-type` to align the spacing between the configuration attribute types.
+
+## Change in Configurator Attribute Drop-Down Component
+
+* `padding` set to 1rem on `%cx-configurator-attribute-drop-down` for `cx-configurator-attribute-quantity` selector to define the spacing between the drop-down attribute type, and the quantity counter.
+
+## Change in Configurator Attribute Checkbox List Component
+
+* `padding` set to 1rem on `%cx-configurator-attribute-checkbox-list` to define the spacing between the checkbox-list attribute type, and the quantity counter.
+
+## Change in Configurator Attribute Radio Button Component
+
+* `padding` set to 1rem on `%cx-configurator-attribute-radio-button` to define the spacing between the radio-button attribute type, and the quantity counter.
+
+## Change in Configurator Previous Next Button Component
+
+* `padding` set to 16px on `%cx-configurator-previous-next-buttons` to align the spacing between the configuration form and the bottom of the configuration.
+
+## Change in Configurator Price Summary Component
+
+* `padding` set to 16px on `%cx-configurator-price-summary` for cx-total-summary selector to align the spacing.
+
+## Change in Configurator Footer Container Mixin
+
+* `padding` set to 16px on `%cx-configurator-footer-container` mixin to align the spacing between the price summary and add-to-cart button.
+
+## Change in Configurator Required Error Message Mixin
+
+* `padding` set to 5px on `%cx-configurator-required-error-msg` mixin to add the spacing at the end of the cx-icon selector.
+
+## Changes in Configurator Overview Form Component
+
+* `padding` set to 0px on `%cx-configurator-overview-form` to prevent adding any spacing to the configuration overview form.
```

Comment:
What does that mean exactly?

Reply:
'to fix inconsistent spacings in the configuration overview form'

Reply:
Done!